src/smart-components/request/request-detail/request.js

```diff
@@ -59,7 +59,7 @@ export const Request = ({ item, isExpanded, toggleExpand }) => {
               id={ `request-${request.id}-request-comment` }
               to={ {
                 pathname: routes.request.addComment,
-                search: `?request=${request.id}`
+                search: useLocation().search
```

Comment:
the hook should be used on the top of the component. You might get inaccurate values.